diff --git a/node_modules/@blocknote/xl-ai/dist/server-CafGbRLa.js b/node_modules/@blocknote/xl-ai/dist/server-CafGbRLa.js
index 7fa413f..204a6e3 100644
--- a/node_modules/@blocknote/xl-ai/dist/server-CafGbRLa.js
+++ b/node_modules/@blocknote/xl-ai/dist/server-CafGbRLa.js
@@ -476,7 +476,7 @@ async function j(e, t, n, r) {
 		if (d.reason instanceof D) f = d.reason;
 		else if (!t.error) throw Error("Unexpected: no ChunkExecutionError but also no chat.error (network error?)");
 	}
-	let p = !1, m = Array.from(s.values().filter((e) => e.complete));
+	let p = !1, m = Array.from(s.values()).filter((e) => e.complete);
 	return m.forEach((e, n) => {
 		let r = e.toolCallId === f?.chunk.metadata.toolCallId;
 		r && (p = !0), p ? t.addToolOutput({
